calico/reference/felix/configuration.mdx

```diff
@@ -198,7 +194,13 @@ See the [HOWTO guide](../../operations/ebpf/enabling-ebpf.mdx) for step-by step
 | BPFHostConntrackBypass / <br/> FELIX_BPFHostConntrackBypass                         | Controls whether to bypass Linux conntrack in BPF mode for workloads and services.                                                                                                                                                                                                                                                                                                | true,false            | true    |
+| BPFEnforceRPF / <br/>  FELIX_BPFEnforceRPF                                          | Controls RPF on all host interfaces with BPF programs regardless of what is the per-interfaces or global setting. To prevent spoofing, we also enforce it in IPv6 mode. Possible values are `Disabled`, `Strict` or `Loose`.                                                                                                                                                      | string                | "Loose" |
+
 | BPFPolicyDebugEnabled / <br/> FELIX_BPFPOLICYDEBUGENABLED                           | In eBPF dataplane mode, Felix records detailed information about the BPF policy programs, which can be examined with the calico-bpf command-line tool.                                                                                                                                                                                                                            | true, false           | true    |
+| BPFForceTrackPacketsFromIfaces / <br/> FELIX_BPFFORCETRACKPACKETSFROMIFACES         | In BPF mode, forces traffic from these interfaces to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Should only be used for interfaces that are not used for the Calico fabric.  For example, a docker bridge device for non-Calico-networked containers. [Default: docker+]                                      | string                | "docker+" |
+| BPFDisableGROForIfaces / <br/> FELIX_BPFDISABLEGROFORIFACES                         | A regular expression that controls which interfaces Felix should disable the Generic Receive Offload [GRO] option. It should not match the workload interfaces (usually named cali...).                                                                                                                                                                                           | string                | ""      |
+| BPFExcludeCIDRsFromNAT / <br/> FELIX_BPFEXCLUDECIDRSFROMNATIS                       | A list of CIDRs that are to be excluded from NAT resolution so that host can handle them. A typical use case is node local DNS cache. Set it to your kube-dns service IP if you want to use node local DNS cache.                                                                                                                                                                  | string                | ""      |
```

Comment:
```suggestion
| BPFExcludeCIDRsFromNAT / <br/> FELIX_BPFEXCLUDECIDRSFROMNATIS                       | A list of CIDRs that are to be excluded from NAT resolution so that the host can handle them. A typical use case is node local DNS cache. Set it to your kube-dns service IP if you want to use node local DNS cache.                                                                                                                                                                  | string                | ""      |
```